lib/v2/uestc/news.js

```diff
@@ -0,0 +1,57 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const dateRegex = /(20\d{2}).(\d{2})-(\d{2})/;
+
+const baseUrl = 'https://news.uestc.edu.cn';
+
+const map = {
+    academy: '/?n=UestcNews.Front.CategoryV2.Page&CatId=66',
+    culture: '/?n=UestcNews.Front.CategoryV2.Page&CatId=67',
+    announcement: '/?n=UestcNews.Front.CategoryV2.Page&CatId=68',
+};
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type || 'announcement';
+    const pageUrl = map[type];
+    if (!pageUrl) {
+        throw new Error('type not supported');
+    }
+
+    const response = await got({
+        method: 'get',
+        url: baseUrl + pageUrl,
+        headers: {
+            Referer: baseUrl,
+        },
+    });
```

Comment:
Use either one convention, but not both
https://github.com/DIYgod/RSSHub/blob/8e334b9b0ec4356befb3bb620d5102c301a7eea9/lib/v2/uestc/gr.js#L9